tests/test_eco_news_page.py

```
@@ -0,0 +1,17 @@
+from pages.base import BasePage
+
+
+@pytest.mark.eco_news_filter_by_tags
+def test_eco_news_filter_by_tags(login_driver):
+    page = EcoNewsListPage(login_driver)
+    page.open_page_by_link("https://www.greencity.cx.ua/#/greenCity/news")
+    tags = ["NEWS", "EVENTS", "EDUCATION", "INITIATIVES", "ADS"]
+    # step_1
+    for tag in tags:
+        page.click_tag_filter(tag)
+        assert page.is_tag_in_list(tag) == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `page.is_tag_in_list(tag):` for truth checks

Replace with `page.is_tag_in_list(tag)`

(E712)